packages/webpack/chunk-loading-webpack-plugin/src/StartupChunkDependenciesPlugin.ts

```diff
@@ -62,6 +64,20 @@ export class StartupChunkDependenciesPlugin {
           : globalChunkLoading;
         return chunkLoading === this.chunkLoading;
       };
+      const handler = (chunk: Chunk, runtimeRequirements: Set<string>) => {
```

Comment:
I don't think it make sense to have this in the `StartupChunkDependenciesPlugin.ts`:

1. This is copied from webpack. We should keep it as it.
2. This should be placed within the same plugin that injecting the RuntimeModule. This could avoid making the two plugins coupled.